content/manager/broker-security.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: bt_wiki
+title: Broker Security (RabbitMQ)
+category: Manager
+draft: false
+weight: 1000
+
+---
+
+Cloudify uses RabbitMQ as its broker, and supports configurable security.
+
+# Authentication
+
+When bootstrapping, the cloudify manager must be provided with credentials for rabbit. These will use default values if not overridden in the inputs.
+
+## Username
+
+It is suggested that you change the username to something other than the default. It is recommended that you use only upper and lower case letters and numbers for the username.
+
+The username can be set using the rabbitmq_username input to the 3.3 manager blueprint.
+
+## Password
+
+It is recommended that you set the password to something strong that is known only to those who are authorized.
+
+It is recommended that the password is comprised of only ASCII characters, excluding quotes, line feeds, and dollar signs, as these may cause the password to be set incorrectly during the bootstrap.
+
+The password should be set using the rabbitmq_password input to the 3.3 manager blueprint.
+
+## With external broker
+
+If you are using an external broker you must have correctly configured the user on the external RabbitMQ broker. This user must have full permissions on the root (/) vhost.
+
+No changes of configuration of the external broker will be performed.
+
+# Network Encryption
+
+Communications with the broker can be secured (with some exceptions, see below) using SSL/TLS.
+
+A certificate from a certificate authority is not required, as the provided public certs will be 'pinned' (they will be the only certificate valid for connections to the broker).
+
+Currently you will likely need to create the manager with a specified IP in order to appropriately create the certificate.
+
+## Generating a certificate
+
```

Comment:
um? You're saying that to use the cert a user must create a key based on an already running manager's IP address? If we bootstrap on openstack, for instance, and an ip is automatically assigned, how will that work?